cmd/snap-update-ns/entry_test.go

```diff
@@ -106,9 +108,21 @@ func (s *entrySuite) TestXSnapdGid(c *C) {
 	c.Assert(gid, Equals, uint64(0))
 
 	// Group is parsed from the x-snapd-group= option.
-	nogroupGid, err := osutil.FindGid("nogroup")
-	c.Assert(err, IsNil)
-	e = &mount.Entry{Options: []string{"x-snapd.gid=nogroup"}}
+	var nogroup string
+	var nogroupGid uint64
+	// try to find a suitable 'nogroup' like group
```

Comment:
(nitpick) Maybe expand the comment a tiny bit? Something like: `try to find a suitable 'nogroup' like group, e.g. arch uses "nobody"`